scripts/scil_convert_rgb.py

```diff
@@ -66,9 +67,7 @@ def main():
     elif original_im.ndim == 3:
         converted_im_float = decfa_to_float(original_im)
 
-        converted_data = converted_im_float.get_fdata()
-        converted_data_int = converted_data.astype("uint8")
-
+        converted_data_int = get_data_as_mask(converted_im_float)
```

Comment:
Since an RGB is not a mask, I think this is (1) confusing and (2) will likely crash due to the fact that RGB contain more than one value. It should stay like before.

Reply:
Ok will switch back